nginx-unprivileged/Dockerfile

```
@@ -12,7 +12,7 @@ ARG RELEASE
 ARG NGINX_UID=999
 ARG NGINX_GID=999
 
-ENV SUMMARY="UBI based Unprivileged NGINX" \
+ENV SUMMARY="Unprivileged NGINX" \
     DESCRIPTION="nginx is an HTTP and reverse proxy server, a mail proxy server, and a generic TCP/UDP proxy server"
```

Let's clear this while we have an opportunity:
```suggestion
ENV SUMMARY="Unprivileged NGINX" \
    DESCRIPTION="NGINX is an HTTP and reverse proxy server, a mail proxy server, and a generic TCP/UDP proxy server"
```